docs/JSDoc/Interfaces.js.html

```diff
@@ -641,27 +640,22 @@ <h1><a href="index.html" class="link">Situm</a></h1>
   outdoorLocationOptions,
   beaconFilters,
   smallestDisplacement,
-  realtimeUpdateInterval
-}
+  realtimeUpdateInterval,
+  autoEnableBleDuringPositioning,
+};
 
-module.exports = LocationOptions
+module.exports = LocationRequest;
 
 /**
  * @name
- * LocationRequest
+ * LocationOptions
  * @description
  * A data object that contains parameters for the location service, LocationManager.
- * @type {array}
- * @property {Building} building 0 - Building on which the positioning will be started
- * @property {LocationOptions} locationOptions 1 - Location options.
+ * @deprecated
```

Comment:
Eu linkearia aqui a clase de LocationRequest con un "This class is deprecated, instead use {@link LocationRequest}"